.github/workflows/winget.yml

```diff
@@ -0,0 +1,13 @@
+name: Publish to WinGet
+on:
+  release:
+    types: [released]
```

Comment:
![image](https://user-images.githubusercontent.com/83997633/219923053-5ca8e14e-219e-4ca1-a151-367bbff92940.png)

Reply:
@fredemmott the above screenshot is from https://docs.github.com/webhooks-and-events/webhooks/webhook-events-and-payloads?actionType=released#release.